<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1588</width>
    <height>1123</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTabWidget" name="tabWidget1">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>80</y>
     <width>1091</width>
     <height>961</height>
    </rect>
   </property>
   <property name="currentIndex">
    <number>4</number>
   </property>
   <widget class="QWidget" name="tab_6">
    <attribute name="title">
     <string>自动化测试</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget_2">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>100</y>
       <width>721</width>
       <height>331</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="retry_cpu_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_3">
     <property name="geometry">
      <rect>
       <x>320</x>
       <y>460</y>
       <width>711</width>
       <height>361</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="retry_mem_layout"/>
    </widget>
    <widget class="QPushButton" name="start_script_auto">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>190</y>
       <width>111</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>脚本重放</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retry_script_first">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>录制前/后置脚本</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retry_script_cycle">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>录制循环脚本</string>
     </property>
    </widget>
    <widget class="QPushButton" name="retry_script_save">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>100</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>保存录制脚本</string>
     </property>
    </widget>
    <widget class="QPushButton" name="delete_auto_script">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>190</y>
       <width>121</width>
       <height>41</height>
      </rect>
     </property>
     <property name="text">
      <string>删除脚本</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_15">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>脚本名称</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_16">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>250</y>
       <width>81</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>重放次数</string>
     </property>
    </widget>
    <widget class="QListWidget" name="retry_script_list">
     <property name="geometry">
      <rect>
       <x>5</x>
       <y>311</y>
       <width>271</width>
       <height>291</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="retry_script_name">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>270</y>
       <width>131</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLineEdit" name="retry_script_time">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>270</y>
       <width>111</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>5</string>
     </property>
    </widget>
    <widget class="QPushButton" name="open_script_list">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>140</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>打开脚本目录</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab">
    <attribute name="title">
     <string>性能测试</string>
    </attribute>
    <widget class="QWidget" name="horizontalLayoutWidget">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>40</y>
       <width>801</width>
       <height>261</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="fps_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_4">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>330</y>
       <width>801</width>
       <height>281</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="cpu_layout"/>
    </widget>
    <widget class="QWidget" name="horizontalLayoutWidget_5">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>650</y>
       <width>801</width>
       <height>271</height>
      </rect>
     </property>
     <layout class="QHBoxLayout" name="mem_layout"/>
    </widget>
    <widget class="QLabel" name="label_10">
     <property name="geometry">
      <rect>
       <x>410</x>
       <y>10</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_11">
     <property name="geometry">
      <rect>
       <x>420</x>
       <y>310</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU数据</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_12">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>620</y>
       <width>71</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>MEM数据</string>
     </property>
    </widget>
    <widget class="QPushButton" name="performance_start_mem_test">
     <property name="geometry">
      <rect>
       <x>900</x>
       <y>760</y>
       <width>101</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>内存测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="performance_start_fps_test">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>110</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>FPS测试</string>
     </property>
    </widget>
    <widget class="QPushButton" name="performance_start_cpu_test">
     <property name="geometry">
      <rect>
       <x>910</x>
       <y>430</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>CPU测试</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_5">
    <attribute name="title">
     <string>日志工具</string>
    </attribute>
    <widget class="QTextEdit" name="show_log">
     <property name="geometry">
      <rect>
       <x>240</x>
       <y>90</y>
       <width>751</width>
       <height>741</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="start_get_Log">
     <property name="geometry">
      <rect>
       <x>280</x>
       <y>50</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>开始获取日志</string>
     </property>
    </widget>
    <widget class="QPushButton" name="stop_get_Log">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>50</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>停止获取日志</string>
     </property>
    </widget>
    <widget class="QPushButton" name="clear_log">
     <property name="geometry">
      <rect>
       <x>550</x>
       <y>50</y>
       <width>113</width>
       <height>32</height>
      </rect>
     </property>
     <property name="text">
      <string>清空日志</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="filter_log_key">
     <property name="geometry">
      <rect>
       <x>810</x>
       <y>50</y>
       <width>181</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_7">
    <attribute name="title">
     <string>崩溃日志</string>
    </attribute>
    <widget class="QPushButton" name="get_crash_log">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>获取crash日志</string>
     </property>
    </widget>
    <widget class="QPushButton" name="export_crash_log">
     <property name="geometry">
      <rect>
       <x>400</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>导出日志</string>
     </property>
    </widget>
    <widget class="QPushButton" name="remove_crash_log">
     <property name="geometry">
      <rect>
       <x>670</x>
       <y>70</y>
       <width>191</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>删除日志</string>
     </property>
    </widget>
    <widget class="QTableWidget" name="crash_tab_list">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>170</y>
       <width>861</width>
       <height>641</height>
      </rect>
     </property>
     <attribute name="horizontalHeaderCascadingSectionResizes">
      <bool>true</bool>
     </attribute>
    </widget>
   </widget>
   <widget class="QWidget" name="tab_2">
    <attribute name="title">
     <string>应用管理</string>
    </attribute>
    <widget class="QLabel" name="page2_show_result">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>160</y>
       <width>511</width>
       <height>441</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>170</y>
       <width>551</width>
       <height>301</height>
      </rect>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
    <widget class="QTableWidget" name="app_table_list">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>110</y>
       <width>911</width>
       <height>731</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>35</y>
     <width>101</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>应用进程</string>
   </property>
  </widget>
  <widget class="QLabel" name="device_id">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>40</y>
     <width>131</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QComboBox" name="device_app_list">
   <property name="geometry">
    <rect>
     <x>720</x>
     <y>20</y>
     <width>381</width>
     <height>51</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="phone_show">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>120</y>
     <width>341</width>
     <height>771</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="phone_menu">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>920</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>菜单键</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phone_home">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>920</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>home键</string>
   </property>
  </widget>
  <widget class="QPushButton" name="phone_back">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>920</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>返回键</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>tabWidget1</sender>
   <signal>currentChanged(int)</signal>
   <receiver>tabWidget1</receiver>
   <slot>setCurrentIndex(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>327</x>
     <y>293</y>
    </hint>
    <hint type="destinationlabel">
     <x>327</x>
     <y>293</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
